[Android] Make db size configurable (#98)
react-native-community/react-native-async-storage: README.md

```

### Advanced

> See docs for [api and more examples](docs/API.md) or [advanced usages](docs/advanced).

Reviewer: Is the link here still correct?

> See docs for [api and more examples](docs/API.md), [brownfield integration guide](docs/advanced/BrownfieldIntegration.md) and [increased storage size on Android](docs/advanced/IncreaseDbSize.md) .

Reviewer 2: We can point to `advanced` docs directly

```suggestion
See docs for [api and more examples](docs/API.md) or [advanced usages](docs/advanced).
```
